packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -146,14 +172,23 @@ export const PaymentForm = ({
     ));
   }
 
-  return (
-    <Form
-      data-testid="paymentForm"
-      validator={validator}
-      onSubmit={onSubmit}
-      className="payment"
-      {...{ onChange }}
-    >
+  const paymentSource = hasExistingCard ? (
+    <div className="card-details" data-testid="card-details">
+      <Localized
+        id="sub-update-card-ending"
+        vars={{
+          last: (customer as Customer).last4!,
```

Comment:
`hasExistingCard` contains a check for `customer` and `last4` so this assertion should be ok.

Also, I don't know why it's `last` and not `last4` where, but I did check the ftl file and it is `last` in there.

Reply:
It's fun to be inconsistent I guess?